RecoTauTag/RecoTau/test/rerunMVAIsolationOnMiniAOD_Phase2.py

```diff
@@ -0,0 +1,106 @@
+import FWCore.ParameterSet.Config as cms
+
+process = cms.Process("rerunMVAIsolationOnMiniAODPhase2")
+
+process.load('Configuration.Geometry.GeometryExtended2026D41Reco_cff')
+process.load("Configuration.StandardSequences.FrontierConditions_GlobalTag_cff")
+from Configuration.AlCa.GlobalTag import GlobalTag
+process.GlobalTag = GlobalTag(process.GlobalTag, '106X_upgrade2023_realistic_v2')
+process.load("Configuration.StandardSequences.MagneticField_cff")
+
+#process.Tracer = cms.Service("Tracer")
+process.load("FWCore.MessageService.MessageLogger_cfi")
+process.MessageLogger.cerr.FwkReport.reportEvery = 1000
+process.maxEvents = cms.untracked.PSet(
+    input = cms.untracked.int32(10000)
+)
+
+process.source = cms.Source("PoolSource",
+    fileNames = cms.untracked.vstring(
+        #'/store/mc/PhaseIIMTDTDRAutumn18MiniAOD/QCD_Pt-15To7000_TuneCP5_Flat_14TeV-pythia8/MINIAODSIM/PU200_103X_upgrade2023_realistic_v2-v1/40000/FFE6B9AD-6109-FA47-9273-24C908EC90EE.root',
+        '/store/mc/PhaseIIMTDTDRAutumn18MiniAOD/VBFHToTauTau_M125_14TeV_powheg_pythia8_correctedGridpack/MINIAODSIM/PU200_103X_upgrade2023_realistic_v2-v1/120000/2AD029AA-54C4-3D44-B934-24F19454A6FD.root',
+    )
+)
+
+process.TFileService = cms.Service("TFileService",
+    #fileName = cms.string('output_QCD_Phase2.root')
+    fileName = cms.string('output_VBFHToTauTau_Phase2.root')
+)
+
+from RecoTauTag.RecoTau.TauDiscriminatorTools import noPrediscriminants
+### Load PoolDBESSource with mapping to payloads
+process.load('RecoTauTag.Configuration.loadRecoTauTagMVAsFromPrepDB_cfi')
+
+from RecoTauTag.RecoTau.PATTauDiscriminationByMVAIsolationRun2_cff import *
+process.rerunDiscriminationByIsolationMVADBnewDMwLTPhase2raw = patDiscriminationByIsolationMVArun2v1raw.clone(
+    PATTauProducer = cms.InputTag('slimmedTaus'),
```

Comment:
This is a test config, but the same suggestion of dropping the type specs can still hold (here and below)